controllers/managedocs_controller.go

```diff
@@ -760,9 +760,10 @@ func (r *ManagedOCSReconciler) reconcileAlertmanagerConfig() error {
 		for {
 			alertingAddress, found := r.addonParamSecret.Data[notificationEmailKeyPrefix+fmt.Sprintf("-%v", i)]
 			i++
-			if found {
-				alertingAddressList = append(alertingAddressList, string(alertingAddress))
-			} else {
+			alertingAddressAsString := string(alertingAddress)
+			if found && alertingAddressAsString != "" {
+				alertingAddressList = append(alertingAddressList, alertingAddressAsString)
+			} else if !found {
```

Comment:
Why check found twice? 
```suggestion
			if found  {
				alertingAddressAsString := string(alertingAddress)
				if alertingAddressAsString != nil {
					alertingAddressList = append(alertingAddressList, alertingAddressAsString)
				}
			} else {
```

Reply:
Done